exercises/list-ops/canonical-data.json

```diff
@@ -57,14 +57,14 @@
           "description": "empty list",
           "property": "filter",
           "list": [],
-          "function": "value modulo 2 == 1",
+          "function": "(x) -> x modulo 2 == 1",
```

Comment:
@ErikSchierboom would "list" and "function" be considered part of "input" in a future PR?

Reply:
Yes, they would!